flatliners/baseflatliner.py

```diff
@@ -50,4 +50,8 @@ def create_cache_dict(maxsize=2000, ttl=900):
         # ttl determines the time to live for each element in the dictionary, so
         # $(ttl) seconds after the element is added, it will be marked for deletion
         # if the element is updated, the timer resets.
+        if not ttl:
```

Comment:
But isn't `ttl` defaulted to 900?
So if you dont provide it as an argument, it'll still return a LRUCache.
You have to explictly pass `None`, which feels awkward